website/content/en/docs/best-practices/pod-security-standards.md

```diff
@@ -54,6 +54,7 @@ Please, do **not** use this field if you are looking to build Operators that wor
 
 **On Reconciliations, such as code implementation in Go:**
 
+**Note:** *if you are setting the `RunAsNonRoot` value to `true` in the `SecurityContext` you will also need to specify the `RunAsUser` value to set the user to a numeric user. In this implementation it sets the user uid to `1000`.*
```

Comment:
Hi @everettraven, 

It does not seem true. See that the Operator Pod/containers also are with RunAsNonRoot and are working fine: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/config/manager/manager.yaml#L27-L30 (we are checking if it was deployed and if the reconciliation work successfully on the tests right?)

The runAsUser in the Memcached example shows required because of the image itself that requires an user to be executed. Then if we look at the documentation https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted: 

> Containers must not set runAsUser to 0

So, I think we can update it with something like follows:
```suggestion
**Note:**  If your container image requires a user to run then, ensure that you do not use the value 0 for `runAsUser`. Note that user 0 is the root and that would mean your Pod/container is not restricted. 
```

Reply:
/hold

Reply:
That is a good point, thanks for bringing that up @camilamacedo86 ! I will make some changes in a bit and update the PR.

Reply:
This should be resolved as of commits https://github.com/operator-framework/operator-sdk/pull/5904/commits/abaae3623479221dc20641123e7c8908c111a41a and https://github.com/operator-framework/operator-sdk/pull/5904/commits/d8d885ff422341a07d8aec2623f883661189346f